app/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -45,15 +39,10 @@ protected void onCreate(Bundle savedInstanceState) {
         appBarConfiguration = new AppBarConfiguration.Builder(navController.getGraph()).build();
         NavigationUI.setupActionBarWithNavController(this, navController, appBarConfiguration);
 
-        getSupportActionBar().hide();
-        binding.fab.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                Snackbar.make(view, "Replace with your own action", Snackbar.LENGTH_LONG)
-                        .setAnchorView(R.id.fab)
-                        .setAction("Action", null).show();
-            }
-        });
+        Objects.requireNonNull(getSupportActionBar()).hide();
+        binding.fab.setOnClickListener(view -> Snackbar.make(view, "Replace with your own action", Snackbar.LENGTH_LONG)
```

Comment:
I know this was already there, but is this code necessary? It doesn't need to be fixed in this PR (which is mostly fixing warnings), but if it is not needed, it should be removed.